docs/guides/composition-revisions-example.md

```diff
@@ -0,0 +1,302 @@
+```shell
+kubectl get compositionrevisions -o=custom-columns=NAME:.metadata.name,REVISION:.spec.revision,LABEL:.metadata.labels.channel
+```
+Expected Output:
+```shell
+NAME                                    REVISION   LABEL
+myvpcs.aws.example.upbound.io-ad265bc   1          dev
+```
+
+The label `dev` is automatically created from the Composition.
+
+## Create Composite Resources
+
+### Default update policy
+Create an XR without a `compositionUpdatePolicy` defined. The update policy is `Automatic` by default:
+```yaml
+apiVersion: aws.example.upbound.io/v1alpha1
+kind: MyVPC
+metadata:
+  name: vpc-auto
+spec:
+  id: vpc-auto
+```
+Expected Output:
+```shell
+myvpc.aws.example.upbound.io/vpc-auto created
+``` 
+
+### Manual update policy
+Create an Composite Resource with `compositionUpdatePolicy: Manual` and `compositionRevisionRef`.
+```yaml
+apiVersion: aws.example.upbound.io/v1alpha1
+kind: MyVPC
+metadata:
+  name: vpc-man
+spec:
+  id: vpc-man
+  compositionUpdatePolicy: Manual
+  compositionRevisionRef:
+    name: myvpcs.aws.example.upbound.io-ad265bc
+```
+
+Expected Output:
+```shell
+myvpc.aws.example.upbound.io/vpc-man created
+``` 
+
+
+### Using a selector
+Create an XR with a `compositionRevisionSelector` of `channel: dev`:
+```yaml
+apiVersion: aws.example.upbound.io/v1alpha1
+kind:  MyVPC
+metadata:
+  name: vpc-dev
+spec:
+  id: vpc-dev
+  compositionRevisionSelector:
+    matchLabels:
+      channel: dev
+```
+Expected Output:
+```shell
+myvpc.aws.example.upbound.io/vpc-dev created
+``` 
+
+Create an XR with a `compositionRevisionSelector` of `channel: staging`:
+```yaml
+apiVersion: aws.example.upbound.io/v1alpha1
+kind: MyVPC
+metadata:
+  name: vpc-staging
+spec:
+  id: vpc-staging
+  compositionRevisionSelector:
+    matchLabels:
+      channel: staging
+```
+
+Expected Output:
+```shell
+myvpc.aws.example.upbound.io/vpc-staging created
+``` 
+
+## Verify the Composite Resources
+Verify the Composite Resource with the label `channel: staging` doesn't have a `REVISION`.  
+All other XRs have a `REVISION` matching the created Composition Revision.
+```shell
+kubectl get composite -o=custom-columns=NAME:.metadata.name,SYNCED:.status.conditions[0].status,REVISION:.spec.compositionRevisionRef.name,POLICY:.spec.compositionUpdatePolicy,MATCHLABEL:.spec.compositionRevisionSelector.matchLabels
+```
+Expected Output:
+```shell
+NAME          SYNCED   REVISION                                POLICY      MATCHLABEL
+vpc-auto      True     myvpcs.aws.example.upbound.io-ad265bc   Automatic   <none>
+vpc-dev       True     myvpcs.aws.example.upbound.io-ad265bc   Automatic   map[channel:dev]
+vpc-man       True     myvpcs.aws.example.upbound.io-ad265bc   Manual      <none>
+vpc-staging   False    <none>                                  Automatic   map[channel:staging]
+``` 
+
+{{< hint "note" >}}
+The `vpc-staging` XR label doesn't match any existing Composition Revisions.
+{{< /hint >}}
```

Comment:
It looks like this is not rendering as expected.

<img width="933" alt="Screen Shot 2022-12-15 at 14 49 27" src="https://user-images.githubusercontent.com/9900707/207852015-732d9b5d-d4b0-4acb-9311-eed5b8fa5f87.png">

Reply:
this will be fixed on the docs site, please see the `Hints and alert boxes` chapter [here](https://docs.crossplane.io/master/contributing/docs/#code-of-conduct).